app/policies/application_policy.rb

```diff
@@ -1,8 +1,35 @@
-class ApplicationPolicy
-  attr_reader :user, :record
+class ApplicationPolicy < ActionPolicy::Base
+  authorize :user, allow_nil: true
 
-  def initialize(user, record)
-    @user = user
-    @record = record.to_model
+  def index?
+    false
+  end
+
+  def show?
```

Comment:
I think we do not these methods because these actions are unpermitted by default.